DEVX-1712: First tutorial for basic Kafka - console consumer and producer (#339)
confluentinc/kafka-tutorials: _includes/tutorials/console-consumer-produer-basic/kafka/markup/dev/produce-topic.adoc

To produce your first record into Kafka, open a terminal window an run the following command:

+++++
<pre class="snippet"><code class="shell">{% include_raw tutorials/console-consumer-produer-basic/kafka/code/tutorial-steps/dev/console-producer.sh %}</code></pre>
+++++

> The producer will start and wait for you to enter input.  Each line represents one record and to send it you'll hit the enter key.  If you type multiple words then hit enter, the entire line is considered one record.

Reviewer: Is it worth explaining keys & separators, for produce?

Author: Yeah, but I think it's covered in _includes/tutorials/console-consumer-produer-basic/kafka/markup/dev/produce-topic-keys-values.adoc, is the coverage there enough

> To produce your first record into Kafka, open a terminal window an run the following command to get an open shell on your docker container:

Reviewer 2: ```suggestion
To produce your first record into Kafka, open another terminal window and run the following command to open a second shell on the broker container:
```

+++++
<pre class="snippet"><code class="shell">{% include_raw tutorials/console-consumer-produer-basic/kafka/code/tutorial-steps/dev/open-docker-shell.sh %}</code></pre>
+++++

> Then run the folling command to start a console producer:

Reviewer 2: ```suggestion
From inside the second terminal on the broker container, run the following command to start a console producer:
```
